cmd/pebble/cmd_run.go

```diff
@@ -181,3 +185,23 @@ out:
 
 	return d.Stop(ch)
 }
+
+// ensurePebbleDir ensures that the pebble directory exists and is a directory.
+// If the directory doesn't exist, create it.
+func ensurePebbleDir(pebbleDir string) error {
+	st, err := os.Stat(pebbleDir)
+	if err != nil {
+		if !os.IsNotExist(err) {
+			return fmt.Errorf("cannot stat pebble directory: %w", err)
+		}
+		err = os.MkdirAll(pebbleDir, 0755)
+		if err != nil {
+			return fmt.Errorf("cannot create pebble directory: %w", err)
+		}
+		return nil
+	}
+	if !st.IsDir() {
+		return fmt.Errorf("%q is not a directory", pebbleDir)
+	}
```

Comment:
The content of this function looks nice, but it should live inside the osutil package next to similar functions we have there. It's also worth checking out to see if we don't have this there already.

Reply:
I realized I can just use `os.MkdirAll` directly, as that has no effect if the directory already exists. Updated to that.

Side node: the `osutil.IsX` functions are a bit weird to me, because they don't return an error if they can't stat. It seems like there's a difference between successfully stat-ed (directory or otherwise) and error stat-ing due to permission denied or whatever.

Reply:
Interesting.. we should check to see if that causes actual issues in call sites, or if they're mainly using the positive side of it to mean "Yeah, I'm sure that's a directory.".